src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/RazorCodeDocumentProvider.cs

```diff
@@ -2,10 +2,11 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.CodeAnalysis;
 
-namespace Microsoft.CodeAnalysis.Razor
+namespace Microsoft.VisualStudio.LanguageServices.Razor.Editor
```

Comment:
Moved this type to VisualStudio.LanguageServices.Razor so we don't load the workspaces dll (needed for the lazy import statement).